.circleci/config.yml

```diff
@@ -627,11 +627,102 @@ jobs:
       - image: circleci/python:3.9
         environment:
           TOXENV: py39-wheel-cli
+  #
+  # Python 3.10
+  #
+  py310-core:
+    <<: *common
+    docker:
+      - image: circleci/python:3.10
+    environment:
+      TOXENV: py310-core
+
+  py310-ens:
+    <<: *common
+    docker:
+      - image: circleci/python:3.10
+    environment:
+      TOXENV: py310-ens
+
+  py310-ethpm:
+    <<: *ethpm_steps
+    docker:
+      - image: circleci/python:3.10
+    environment:
+      TOXENV: py310-ethpm
+      # Please don't use this key for any shenanigans
+      WEB3_INFURA_PROJECT_ID: 7707850c2fb7465ebe6f150d67182e22
+
+  py310-integration-goethereum-ipc:
+    <<: *geth_steps
+    docker:
+      - image: circleci/python:3.10
+    environment:
+      TOXENV: py310-integration-goethereum-ipc
+      GETH_VERSION: v1.10.8
+
+  py310-integration-goethereum-http:
+    <<: *geth_steps
+    docker:
+      - image: circleci/python:3.10
+    environment:
+      TOXENV: py310-integration-goethereum-http
+      GETH_VERSION: v1.10.8
+
+  py310-integration-goethereum-ws:
+    <<: *geth_steps
+    docker:
+      - image: circleci/python:3.10
+    environment:
+      TOXENV: py310-integration-goethereum-ws
+      GETH_VERSION: v1.10.8
+
+  # py310-integration-parity-ipc:
```

Comment:
We are in the process of getting rid of all parity support so it is ok to remove these 👍